src/main/docker/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+---
+version: '3.8'
+
+services:
+  collectory:
+    container_name: living-atlas-collectory
+    image: livingatlases/collectory:latest
```

Comment:
This can be changed to atlasoflivingaustralia/collectory:latest
https://hub.docker.com/r/atlasoflivingaustralia/collectory

Reply:
changed